receiver/elasticsearchreceiver/client.go

```diff
@@ -87,11 +88,36 @@ func newElasticsearchClient(settings component.TelemetrySettings, c Config, h co
 // https://www.elastic.co/guide/en/elasticsearch/reference/7.9/cluster-nodes-stats.html#cluster-nodes-stats-api-path-params
 const nodeStatsMetrics = "breaker,indices,process,jvm,thread_pool,transport,http,fs,indexing_pressure,ingest,indices,adaptive_selection,discovery,script,os"
 
+// nodesMetrics is a comma separated list of metrics that will be gathered from Nodes.
+// The available metrics are documented here for Elasticsearch 7.9:
+// https://www.elastic.co/guide/en/elasticsearch/reference/7.9/cluster-nodes-info.html
+const nodesMetrics = ""
```

Comment:
It looks like we are specifically asking for no metrics, because we're only interested in the metadata. Seems worth calling out in the comment.